src/sagemaker/clarify.py

```diff
@@ -255,11 +255,11 @@ def __init__(
                     'label_headers=['cat','dog','fish']' and infer the predicted label to be 'fish.'
 
         Args:
-            label (str or int or list[int]): Optional index or JSONPath location in the model
```

Comment:
Would it be probably be good to keep the `Optional` word here as it's not required parameter.